lib/phoenix/tracker/state.ex

```diff
@@ -89,7 +89,7 @@ defmodule Phoenix.Tracker.State do
   Atomically updates ETS local entry.
   """
   @spec leave_join(t, pid, topic, key, meta) :: t
-  def leave_join(%State{delta: delta} = state, pid, topic, key, meta) do
+  def leave_join(state, pid, topic, key, meta) do
```

Comment:
When merging leave and join I kept the pattern match, so the join part operated on delta from before leave. Totally my bad, sorry!